resources/qml/Menus/MaterialMenu.qml

```diff
@@ -1,8 +1,8 @@
 // Copyright (c) 2018 Ultimaker B.V.
 // Cura is released under the terms of the LGPLv3 or higher.
 
-import QtQuick 2.2
-import QtQuick.Controls 1.1
```

Comment:
Those changes don't break things.

Reply:
It actually did break things. Using Qt 5.8 this now breaks. Jaime is changing them back (he was on 5.8 and got those errors).

In general we should require the oldest version that has all features that we use. Since Qt has proper binary compatibility, we will get all bugfixes from new versions but the older Qt versions are still usable then.